Docs: Document `crossOrigin` option (#61455)
vercel/next.js: docs/03-pages/02-api-reference/03-next-config-js/crossOrigin.mdx

---
title: crossOrigin

> description: Use the `crossOrigin` option to add a crossOrigin tag on the `script` tag for pages. Learn more about it here.

Author: ```suggestion
description: Use the `crossOrigin` option to add a crossOrigin tag on the `script` tags generated by `next/script` and `next/head`.
```
